src/Domains/FieldTypeParameters/AbstractFieldTypeParameter.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Synetic\Migator\Domains\FieldTypeParameters;
+
+abstract class AbstractFieldTypeParameter implements FieldTypeParameterInterface
+{
+    private bool $hasValue = false;
+
+    private int|string|array|null $value;
```

Comment:
wauw, ik vloek waarschijnlijk, maar wat verschilt dit van 'mixed'? Is dit puur zodat je geen object of bool erin kunt gooien?

Reply:
Yes, in het kader van zo strict mogelijk. Array zal zelfs uiteindelijk een StringCollection of EnumCollection kunnen worden waarbinnen we afdwingen dat er alleen maar strings in zitten.